charts/lm-logs/templates/configmap.yaml

```diff
@@ -57,6 +57,7 @@ data:
         {{- end}}
         debug false
         compression gzip
+        log_source lm-logs-fluentd-k8s
```

Comment:
IMO, this value is kind of static - so it would be difficult to know the source cluster.
Similar to argus, take a `cluster name` which user can provide to uniquely identify their cluster.
I would recommend to take cluster name input if we don't already.
Format value like : `ll:fluentd:k8s:<clustername>`
CC: @shubhashri12